src/index.html

```diff
@@ -10,8 +10,204 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    class="form">
+      <fieldset class="form__group">
+        <legend>Personal information</legend>
+
+        <div class="form__field">
+          <label>
+            Surname:
+
+            <input
+              type="text"
```

Comment:
Add a 'name' attribute to the input field for 'Surname'. Example: name="surname"